### DISCUSSION TITLE
Invalid module

### DISCUSSION THREAD

**Discussion by starredev** [pocketbase/pocketbase#3841] [Q&A]
Im trying to require node package in pb_hooks but i received invalid module in --debug mode. 
Are there any examples available?

structure:
node_modules

pb_hooks
   - main.pb.js
  
package.json
pocketbase.exe

```

routerAdd("GET", "/hello/:name", (c) => {

    const speakeasy = require('../node_modules/speakeasy');

    let name = c.pathParam("name")

    return c.json(200, { "message": "Hello " + name })
})
```

**Comment by ganigeorgiev** [comment]
Note that the path is relative to the current working directory and not the `pb_hooks` file (see [Relative paths](https://pocketbase.io/docs/js-overview/#relative-paths)).

But even if the module path is correct, there is no guarantee that it will work since the module you are trying to import is not a pure JS ES5 and relies on core node modules like `crypto`, `util`, etc. (see also [Caveats and limitations](https://pocketbase.io/docs/js-overview/#caveats-and-limitations))
